js/table/tableview.ts

```diff
@@ -376,7 +376,10 @@ export class TableView {
 
             let pass = self.timePass();
             pass = pass === null ? 0 : pass;
-            return runtimeSettings.game.moveTime - pass;
+            const moveTime = appConfig.timeSettings.moveTime !== undefined ?
```

Comment:
you could use construction `appConfig.timeSettings.moveTime | runtimeSettings.game.moveTime`